data/json/recipes/tools/tools_hand.json

```diff
@@ -1090,29 +1090,16 @@
   },
   {
     "type": "recipe",
-    "activity_level": "MODERATE_EXERCISE",
+    "activity_level": "LIGHT_EXERCISE",
```

Comment:
This looks like an error, you shouldn't be messing with washboard crafting in this pr